### PR TITLE
Support system table of mysql

### PR DESCRIPTION
**Pull request by qingxinhome**
## What type of PR is this?

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #4935

## What this PR does / why we need it:
Support some system tables and views under the MySQL database, temporarily ensure that MySQL client tools such as Navicat can successfully connect to matrixone,
Practice:
When the mo service starts, mysql db and information_ Create a series of necessary system tables under schema db, and the table data is temporarily empty.

### CODE REVIEW

**Review by daviszhen — APPROVED** [review]

**Review by reusee — APPROVED** [review]

File: pkg/cnservice/server.go [hunk 1]
```diff
@@ -256,6 +257,9 @@ func (s *service) createMOServer(inputCtx context.Context, pu *config.ParameterU
 	if err := metric.InitSchema(moServerCtx, ieFactory); err != nil {
```
Comment (nnsgmsone) [inline pkg/cnservice/server.go:257]: Does it really make sense to initialize it this way.

File: pkg/util/sysview/sysview.go [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+const (
+	MysqlDBConst       = "mysql"
+	InformationDBConst = "information_schema"
+	sqlCreateDBConst   = "create database if not exists "
+	sqlUseDbConst      = "use "
+)
+
+var (
+	initMysqlSysTables = []string{
```
Comment (nnsgmsone) [inline pkg/util/sysview/sysview.go:34]: This doesn't seem to be a view, but a normal table? Then why is it called view.

**Review by aressu1985 — APPROVED** [review]

File: pkg/cnservice/server.go [hunk 3]
```diff
@@ -257,6 +258,9 @@ func (s *service) createMOServer(inputCtx context.Context, pu *config.ParameterU
...
```
Comment (nnsgmsone) [inline pkg/cnservice/server.go:258]: Is this type of initialization really feasible.

File: pkg/util/sysview/sysview.go (hunk 2 above)
Comment (nnsgmsone) [inline pkg/util/sysview/sysview.go:34]: Obviously it is the table. Why do you call it view.

**Review by nnsgmsone — APPROVED** [review]

File: pkg/cnservice/server.go (hunk 3 above)
Comment (qingxinhome) [reply pkg/cnservice/server.go:258]: I verified that it was OK

**Review by fengttt — COMMENTED** [review]
BVT is seriously lacking.   You need to create a few tables, views, etc, and show that these tables contain correct data.

**Comment by qingxinhome** [comment]
> BVT is seriously lacking. You need to create a few tables, views, etc, and show that these tables contain correct data.

These tables are just to simulate some system tables and views of MySQL and ensure that the navicat tool can connect to the mo, so there is no data for the time being. This has been confirmed with Zhang Xiao. Therefore, the focus of bvt testing is to ensure that these tables and their fields exist. OK, but I can add some bvt tests that show table fields. @fengttt @domingozhang

**Comment by fengttt** [comment]
> > BVT is seriously lacking. You need to create a few tables, views, etc, and show that these tables contain correct data.
> 
> 
> 
> These tables are just to simulate some system tables and views of MySQL and ensure that the navicat tool can connect to the mo, so there is no data for the time being. This has been confirmed with Zhang Xiao. Therefore, the focus of bvt testing is to ensure that these tables and their fields exist. OK, but I can add some bvt tests that show table fields. @fengttt @domingozhang 

So user will not see any data?

Even that, you should have a test, user run select * from ...

**Comment by qingxinhome** [comment]
> > > BVT is seriously lacking. You need to create a few tables, views, etc, and show that these tables contain correct data.
> > 
> > 
> > These tables are just to simulate some system tables and views of MySQL and ensure that the navicat tool can connect to the mo, so there is no data for the time being. This has been confirmed with Zhang Xiao. Therefore, the focus of bvt testing is to ensure that these tables and their fields exist. OK, but I can add some bvt tests that show table fields. @fengttt @domingozhang
> 
> So user will not see any data?
> 
> Even that, you should have a test, user run select * from ...

OK，I understand.

**Review by fengttt — APPROVED** [review]

**Review by nnsgmsone — APPROVED** [review]

**Pull request merged**